lcmgen/emit_csharp.c

```diff
@@ -7,6 +7,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
+#include "../lcm/lcm_version.h"
```

Comment:
I would prefer to add an include to `lcmgen/CMakeLists.txt` and use an `<>` include here. My thinking is that having includes in the `CMakeLists.txt` files makes it easier to trace relationships and dependencies between the various components of LCM and relative paths in includes feel like an antipattern.

What do you think about 633fd74a98d56b18929cc416f0bd0b08f8b6c50e?

Reply:
Worth noting: my example commit 633fd74a98d56b18929cc416f0bd0b08f8b6c50e fails the format check, so you might not want to cherry-pick it as-is

Reply:
Sure. I just went with what `main.c` was already doing.

Also an option is to have[ cmake generate the version.h](https://cmake.org/cmake/help/v3.16/guide/tutorial/index.html#adding-a-version-number-and-configured-header-file). It could make the `LCM_VERSION_STRING` without needing the helper `MACRO_LCM_STRINGIFY` macro. But if you want that, I ask that you implement it. I do not want mess with cmake : )

I left "Allow edits by maintainers" checked. Feel free to literally push the changes you see fit.

Reply:
>I left "Allow edits by maintainers" checked. Feel free to literally push the changes you see fit.

Fair enough, I didn't want to assume but I'm happy to make those changes. Added f88b8a65e2fa54b8bbb1d061546866ab8e542910 to this PR.